packages/components/package.json

```diff
@@ -51,6 +53,7 @@
     "@babel/preset-env": "^7.12.7",
     "@babel/preset-react": "^7.12.7",
     "@babel/preset-typescript": "^7.12.7",
+    "@chanzuckerberg/axe-storybook-testing": "^2.0.0",
```

Comment:
We should add an explicit (dev) dependency on axe-core in this package.

That's because axe-core is a peer dep of axe-storybook-testing. This means axe-storybook-testing expects anyone using it to also have a dependency on axe-core.

Peer deps are overrused, in my opinion, but I still went with it for axe-storybook-testing 😬 .

Reply:
ooh good catch, dunno why npm didn't warn me

Reply:
still dunno what happened earlier, but apparently `story-utils` also needed a higher peer dep for testing-library/react below :-)